iceberg/iceberg-handler/src/test/queries/positive/row_count.q

```diff
@@ -0,0 +1,42 @@
+drop table llap_orders;
```

Comment:
I'm not 100% sure but maybe there can be an edge case when the table is not cleaned up properly. In that case, I would consider using `DROP TABLE IF EXISTS...`

Reply:
Fixed